### PR TITLE
feat: add indication for number of synonyms/antonyms left

### PR DESCRIPTION
**Pull request by MySmilingTurban**
adds indication for number of synonyms or antonyms left
plus confetti and toast to indicate completion of semantics task

### CODE REVIEW

File: src/pages/word/semantics/index.tsx [hunk 1]
```diff
@@ -139,6 +145,34 @@ export default function Semantics() {
...
+    // if all words are found, show confetti
+    if (allWords.every((wordFound) => wordFound)) {
+      setShowConfetti(true);
+    } else {
+      setShowConfetti(false);
+    }
+  }, [currentWord, synonyms, antonyms]);
+
+  const showToastMessage = () => {
+    toast.success(text('SEMANTICS_PRAISE'), {
```
Comment (amitojsingh) [inline src/pages/word/semantics/index.tsx:161]: Do we need toast veer ji? I can see toast is covering the next button. Although, confetti is quite good.

File: src/pages/word/semantics/index.tsx [hunk 2]
```diff
@@ -224,29 +255,39 @@ export default function Semantics() {
...
                     text,
+                    currentWord.antonyms as (string | number)[],
+                    setIsAntonymsDisabled,
                     jumpBoxRef,
                   )
-                }
+                )}
               </Droppable>
             </div>
           </div>
```
Comment (amitojsingh) [inline src/pages/word/semantics/index.tsx:293]: The file is getting too big is it possible if we can divide code in subcomponents/utils/custom hooks?

**Review by amitojsingh — CHANGES_REQUESTED** [review]

File: src/pages/word/semantics/index.tsx (hunk 1 above)
Comment (MySmilingTurban) [reply src/pages/word/semantics/index.tsx:161]: Veerji, I have moved the position of toast to top center, so it doesn't cover any existing items. I think it gives some positive reinforcement that you are on track and what's coming next.

File: src/pages/word/semantics/index.tsx (hunk 2 above)
Comment (MySmilingTurban) [reply src/pages/word/semantics/index.tsx:206]: fixed

**Review by amitojsingh — APPROVED** [review]

**Pull request merged**